pkg/crc/constants/constants.go

```diff
@@ -64,3 +61,13 @@ func EnsureBaseDirExists() error {
 	}
 	return nil
 }
+
+// Validate the given driver is supported or not
+func ValidateDriver(driver string) error {
```

Comment:
Not so sure about the location of this function.

At the moment I am busy with a refactor and it will certainly deal with this. so OK for now.